akka-cluster-typed/src/main/scala/akka/cluster/typed/Cluster.scala

```diff
@@ -169,6 +169,8 @@ abstract class Cluster extends Extension {
   /** Details about this cluster node itself */
   def selfMember: Member
 
+  def selfUniqueAddress: UniqueAddress
```

Comment:
don't think this is needed since `selfMember.uniqueAddress` is the same

Reply:
It's now being used [here](https://github.com/akka/akka/pull/26074/files/218715ff553d5f2d86a81ae65b3f75f8e4339607#diff-3caa31ce0b4591cbf66d419c9d78404eR33), WDYT?

Reply:
which can be replaced by
```
implicit val selfUniqueAddress: SelfUniqueAddress = SelfUniqueAddress(Cluster(system).selfMember.uniqueAddress)
```

Reply:
this is using the typed cluster now for the [above code](https://github.com/akka/akka/pull/26074/commits/eb5772dfbe58d1ea1eb101a424c49a8c12bd6131#diff-5c75e56f6e5f5e050fd15fedf5af6ba0R38)

Reply:
I think we are talking past each other. My point is that I don't want to add extra API in `typed.Cluster` for `selfUniqueAddress` because we already have one entry point for all the self stuff, which is `selfMember`.

If you remove `typed.Cluster. selfUniqueAddress` you can replace the access in DistributedData with `selfMember.uniqueAddress`.

Reply:
Ah, understood, change made, thank you.